openapi.yaml

```diff
@@ -1172,6 +1398,75 @@ paths:
           $ref: "#/components/responses/InternalServerError"
 components:
   schemas:
+    ActionParameters:
+      # TODO: (jam) 2025-03-13 Do we have the actual parameters as property
+      #   attributes of ActionParameters, or do we make this an array of
+      #   key/value types?
+      #   In Juju 3.6 ActionSpec returns a Params struct that says:
+      #     The Params map is expected to conform to JSON-Schema Draft 4 as defined at
+      #     http://json-schema.org/draft-04/schema# (see http://json-schema.org/latest/json-schema-core.html)
+      type: object
+      description: The list of parameters for an action and their associated JSON Schema
+      additionalProperties: true
+    ActionResponse:
+      # TODO: (jam) 2025-03-13 When we discuss actions in docs, etc, we talk
+      #   about actions. Internally running an action creates an Operation, which
+      #   then might have multiple Actions that run off of it. (eg `juju run
+      #   app/0 foo` creatse an operation, where each thing that runs on each
+      #   unit is an action.) However, only `juju exec` creates Operations that
+      #   actually span units. Do we want to expose Operations?
+      type: object
+      description: Information about a given action.
+      properties:
+        tag: # TODO: (jam) 2025-03-13 Figure out what we want instead of tag, plausibly ID?
+          type: string
+        receiver:
```

Comment:
Receiver seems to be a tag type also. Add a todo around specifying what this will look like? Apparently they are machines or units.